### PR TITLE
Create empty ready lists for buffers by default

### PR DESCRIPTION
**Pull request by pavel-shirshov**
**What I did**
Add additional logic to initialize ready list as empty by default.

**Why I did it**
Otherwise the port would be stopped if it doesn't have any buffer configuration

**How I verified it**
Build an image and run on your DUT

**Details if related**

### CODE REVIEW

**Comment by lguohan** [comment]
can you explain why test_port_buffer_rel.py pass in the first place?

**Comment by lguohan** [comment]
do not worry about broadcom and mellanox build failure, it is because sai package header mismatch for now.

File: orchagent/bufferorch.cpp [hunk 1]
```diff
@@ -58,9 +58,20 @@ void BufferOrch::initBufferReadyList(Table& table)
 {
     SWSS_LOG_ENTER();
 
+    // init all ports with an empty list
+    for (const auto& it: gPortsOrch->getAllPorts())
```
Comment (qiluo-msft) [inline orchagent/bufferorch.cpp]: I think gPortsOrch->getAllPorts() will return only one port, the CPU port. Is it what you expected?

File: orchagent/bufferorch.cpp [hunk 2]
```diff
@@ -58,9 +58,20 @@ void BufferOrch::initBufferReadyList(Table& table)
 {
     SWSS_LOG_ENTER();
```
Comment (qiluo-msft) [inline orchagent/bufferorch.cpp]: This function will be called twice, so m_port_ready_list_ref will be overwritten again. Is it a bug?

Comment (qiluo-msft) [reply orchagent/bufferorch.cpp]: I try to fix this in https://github.com/Azure/sonic-swss/pull/590, please help review.

**Pull request merged**